source/_integrations/compensation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Compensation
+description: Instructions on how to integrate compensation sensors into Home Assistant.
+ha_category:
+  - Utility
+  - Sensor
+ha_iot_class: Local Push
+ha_release: '0.117'
+ha_quality_scale: internal
+ha_codeowners:
+  - '@petro31'
+ha_domain: compensation
+---
+
+The `compensation` sensor platform consumes the state from other sensors. It exports the `compensated` value as state and the following values as attributes: `entity_id` and `coefficients`.
+
+## Configuration
+
+To enable the compensation sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: compensation
+    name: Yamaha Zone 1 Volume
+    entity_id: media_player.yamaha_receiver
+    attribute: volume_level
+    unit_of_measurement: dB
+    data_points:
+      - 0.2 -> -80.0
+      - 1.0 -> 0.0
+```
+
+{% configuration %}
+entity_id:
+  description: The entity to monitor.
+  required: true
+  type: string
+data_points:
+  description: "The collection of data point conversions with the format `uncompensated_value -> compensated_value`.  e.g. `1.0 -> 2.1`. The number of required data points is equal to the polynomial `degree` + 1. For example, a linear compensation (with `degree: 1`) requires at least 2 data points."
+  required: true
+  type: list
+name:
+  description: Name of the sensor to use in the frontend.
+  required: false
+  default: Compensation
+  type: string
+attribute:
+  description: Attribute to monitor.
+  required: false
+  type: string
+degree:
+  description: The degree of a polynomial. e.g. Linear compensation (y = x + 3) has 1 degree, Quadratic compensation (y = x<sup>2</sup> + x + 3) has 2 degrees, etc.
```

Comment:
```suggestion
  description: "The degree of a polynomial. e.g., Linear compensation (`y = x + 3`) has 1 degree, Quadratic compensation (`y = x<sup>2</sup> + x + 3`) has 2 degrees, etc."
```

Reply:
will the `` kill the <sup>2</sup>?  That's why I avoided it in the first place.

Reply:
`test<sup>test<\sup>`

Reply:
Looks like it does

Reply:
I left the equations as non-code to ensure the superscript stays.